runtime/altair/src/lib.rs

```diff
@@ -1332,7 +1333,9 @@ mod upgrade {
 			weight += InterestAccrual::upgrade_to_v1();
 			weight += Loans::reference_active_rates();
 			weight += InterestAccrual::remove_unused_rates();
+			weight += InterestAccrual::count_rates();
```

Comment:
did this migration already run on the last runtime upgrade? 

At least the `InterestAccrual::removed_unused_rates()` and the `InterestAccrual::count_rates()` are running on each upgrade. Is this intended and what is their purpose? 

I guess my main point is, that I would like to have them behind some versioned logic or some docs on when they can be removed from the upgrade logic, so that we do not forget them.

Reply:
Moved this to a normal migration in the hooks of the pallet